app/todo_lists/models.py

```diff
@@ -1,14 +1,14 @@
 from app import db
 
+from app.lib.models import Base
 
-class TodoList(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
+
+class TodoList(Base):
```

Comment:
Same comment about using the declared base and your mixin here. Also please add a small doc string.